apps/events/filters.py

```diff
@@ -17,8 +17,25 @@ class EventDateFilter(django_filters.FilterSet):
     event_end__gte = django_filters.DateTimeFilter(field_name='event_end', lookup_expr='gte')
     event_end__lte = django_filters.DateTimeFilter(field_name='event_end', lookup_expr='lte')
     attendance_event__isnull = django_filters.BooleanFilter(field_name='attendance_event', lookup_expr='isnull')
+    is_attendee = django_filters.BooleanFilter(field_name='attendance_event', method='filter_is_attendee')
     event_type = ListFilter()
 
+    def filter_is_attendee(self, queryset, name, value):
+        """
+        Filter events on if a user can attend them or not.
```

Comment:
Should not this say *Filter events based on if a user is attending them or not.*?

Reply:
Yes, that is correct